visualize_test.go

```diff
@@ -449,6 +449,8 @@ func assertCtorEqual(t *testing.T, expected *dot.Ctor, ctor *dot.Ctor) {
 }
 
 func assertCtorsEqual(t *testing.T, expected []*dot.Ctor, ctors []*dot.Ctor) {
+	assert.Len(t, ctors, len(expected))
```

Comment:
Thanks for catching this!

In fact, one step better would probably be making this a `require.Len` so we stop the test immediately and not panic in the following loop if len(ctors) > len(expected)

Reply:
Done.